src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1577,6 +1577,11 @@ internal enum ErrorCode
         ERR_ExprCannotBeFixed = 8385,
         ERR_InvalidObjectCreation = 8386,
         #endregion diagnostics introduced for C# 7.3
+
+        #region diagnostics introduced for C# 8.0
+        ERR_TripleDotNotAllowed = 8401,
```

Comment:
>8401 [](start = 34, length = 4)

Could you please make sure that the range you are using is not conflicting with other features that are already in flight:  Nullable, Patters, DefaultInterfaceImplementations, AsyncStreams, etc

Reply:
that's the purpose of the build we will start in the morning,

---
In reply to: [185362609](https://github.com/dotnet/roslyn/pull/26405#discussion_r185362609) [](ancestors = 185362609)